hack/lib/util.sh

```diff
@@ -202,18 +202,6 @@ kube::util::gen-docs() {
 
   mkdir -p "${dest}/docs/user-guide/kubectl/"
   "${gendocs}" "${dest}/docs/user-guide/kubectl/"
-  mkdir -p "${dest}/docs/admin/"
-  "${genkubedocs}" "${dest}/docs/admin/" "kube-apiserver"
```

Comment:
@thockin I actually reverted this change. These docs are autogenerated, and there seems to be some process that I don't know of that replaces these placeholder files with actual generated docs.

To try and split up changes, I didn't want to touch the autogenerated ones *yet*. Just get rid of the static cruft and fix links to go to the right place